addons/web/static/src/js/views/list/list_editable_renderer.js

```diff
@@ -971,11 +971,15 @@ ListRenderer.include({
         // if we don't have widths yet, computed them
         if (!this.columnWidths) {
             this.columns.forEach((column) => {
-                const width = this._getColumnWidth(column);
-                if (width.match(/[a-zA-Z]/)) { // absolute width with measure unit (e.g. 100px)
-                    column.attrs.absoluteWidth = width;
-                } else { // relative width expressed as a weight (e.g. 1.5)
-                    column.attrs.relativeWidth = parseFloat(width, 10);
+                if (column.attrs.class && column.attrs.class.includes('oe_read_only')) {
```

Comment:
I'm not sure to understand. `oe_read_only` doesn't mean invisible... It's only invisible in edit mode (and there is the `oe_edit_only` mode making it invisible in read mode).
Also includes is probably too weak, as it would return true if there is className `oe_read_only2`